src/lib/format/ProtoMsgTsdFormatter.ts

```diff
@@ -17,6 +17,7 @@ export namespace ProtoMsgTsdFormatter {
         messages: MessageFormatter.IMessageModel[];
         extensions: ExtensionFormatter.IExtensionModel[];
         enums: EnumFormatter.IEnumModel[];
+        wellKnownDeclarations: string[];
```

Comment:
We need Dedicated Types here rather than `string` type

Reply:
Okay got it, thanks. Do you have a suggestion on what those would be? I was modeling this after the `imports` field, since they are roughly equivalent (static strings that simply need to be inserted at the bottom of the file), but this could also be aliased like `type WellKnownDeclaration = string;` if that's preferable.